open_schools_platform/student_management/students/services.py

```diff
@@ -0,0 +1,188 @@
+import uuid
+from typing import Dict
+
+from django.core.exceptions import BadRequest
+from django.db.models import QuerySet
+from phonenumber_field.phonenumber import PhoneNumber
+from rest_framework.exceptions import NotAcceptable, MethodNotAllowed
+
+from open_schools_platform.common.services import model_update, BaseQueryHandler
+from open_schools_platform.common.utils import filter_dict_from_none_values, form_ids_string_from_queryset
+from open_schools_platform.organization_management.circles.models import Circle
+from open_schools_platform.student_management.students.exports import StudentExport
+from open_schools_platform.parent_management.families.models import Family
+from open_schools_platform.parent_management.families.services import add_student_profile_to_family, create_family
+from open_schools_platform.photo_management.photos.models import Photo
+from open_schools_platform.query_management.queries.models import Query
+from open_schools_platform.query_management.queries.services import query_update, create_query
+from open_schools_platform.student_management.students.models import StudentProfile, Student, \
+    StudentProfileCircleAdditional
+from open_schools_platform.student_management.students.selectors import get_student_profile
+from open_schools_platform.user_management.users.models import User
+
+
+def create_student_profile(name: str, age: int = None, user: User = None,
+                           phone: PhoneNumber = None, photo: uuid.UUID = None) -> StudentProfile:
+    if not photo:
+        photo = Photo.objects.create_photo()
```

Comment:
can we create Photo in manager?

Reply:
it is better to create it inside service, because manager for atomic operation - creation of one object